documentation/docs/quick-start.md

```diff
@@ -28,8 +28,8 @@ The diagram details how the concepts fit alongside the PRISM components in a typ
 
 
 
-## An overview of Atala PRISM components
-Atala PRISM consists of core libraries that facilitate typical SSI interactions between [Issuers](/docs/concepts/glossary/#issuer), [Holders](/docs/concepts/glossary/#holder), and [Verifiers](/docs/concepts/glossary/#verifier).
+## An overview of Hyperledger Identus components
```

Comment:
'Atala PRISM' is replaced by `Identus`. See the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0) for reference.